### ISSUE TITLE
Clarifying

### ISSUE THREAD

**Issue by NightlineGT** [snndmaa/solana_escrow_contract#1]
Hey so I've read the description of your project here and your test code and it looks very interesting! I wanna know how it works in even more detail and what's its going to be used for.

So this program that you've written, is it a program for employers and workers can use where they can connect P2P with the security of smart contracts and the blockchain to make sure of one's payment?

**Comment by snndmaa** [comment]
Yes @NightlineGT you have the idea right. Still working on improving it but its also going to work with a mobile app I built to revolutionize work. You can check out the project here [jobGo](https://jobgo.pythonanywhere.com/) although its still in it's early stages.

**Comment by NightlineGT** [comment]
That's awesome man! Im actually just starting to learn Rust but I have a background in verh basic Python and HTML CSS. Idk if I can be much help but if your interested in collaborating we can learn together!

**Comment by snndmaa** [comment]
Yeah for sure I'm willing to collaborate. We can connect on discord and talk further if you'd like.
